crowbar_framework/config/locales/suse_manager_client/en.yml

```diff
@@ -18,10 +18,12 @@ en:
   barclamp:
     suse_manager_client:
       edit_attributes:
-        manager_server: 'SUSE Manager server hostname'
-        manager_server_hint: 'For example susemgr.example.com. You can set a record within the %{link}.'
-        activation_key: 'Activation Key'
-        activation_key_hint: 'For example 1-00112233445566778899aabbccddeeff'
+        bootstrap_script_url: 'URL of the bootstrap script'
+        bootstrap_script_url_hint: |
+          This is the script administrator of SUSE Manager server has
```

Comment:
Sorry, it seems I forgot one: I would add "the" before "administrator" and "SUSE Manager server".

Feel free to ignore and merge, or fix and merge.